src/components/Button.js

```diff
@@ -0,0 +1,22 @@
+import PropTypes from 'prop-types';
+
+function Button(props) {
+  const { buttonName } = props;
+  return (
+    <div>
+      <button type="button">
+        {buttonName}
+      </button>
+    </div>
+  );
+}
+
+Button.propTypes = {
+  buttonName: PropTypes.string,
+};
```

Comment:
Since the button name is required, you can include that in the prop-type definition. i.e. 
`  buttonName: PropTypes.string.isRequired`